bindings/pyroot/pythonizations/test/rdf_filter.py

```diff
@@ -0,0 +1,123 @@
+import unittest
+import ROOT
+import numpy as np
+
+# These imports are specific to test_with_dtypes
+import random 
+import pandas as pd
```

Comment:
I don't think we can use `pandas` here, it's not in our `requirements.txt` file so I don't know whether our CI nodes have it. I may be wrong though